tests/UnitTests/DurableDemographicTests/DurableDemographicTests.cs

```diff
@@ -136,7 +150,8 @@ public async Task HttpStart_InvalidInput_ReturnsNull()
             clientMock.Object,
             mockFunctionContext.Object);
 
-        Assert.IsNull(result);
+        Assert.IsNotNull(result);

```

Comment:
Can the test name be updated to something like:
```suggestion
        HttpStart_InvalidInput_ReturnsInternalServerError()
```